keyboards/pangorin/tan67/keymaps/default/keymap.c

```diff
@@ -0,0 +1,134 @@
+    [_FN2]  = LAYOUT(
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+                _______, _______, _______,                   _______,                            _______, _______, _______, _______, _______
+            ),
+/*
+┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───────┐┌───┐
+│   ││   ││   ││   ││   ││   ││   ││   ││   ││   ││   ││   ││   ││       ││   │
+└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───────┘└───┘
+┌─────┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌─────┐┌───┐
+│     ││   ││   ││   ││   ││   ││   ││   ││   ││   ││   ││   ││   │|     ││   │
+└─────┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└─────┘└───┘
+┌──────┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌─────────┐┌───┐
+│      ││   ││   ││   ││   ││   ││   ││   ││   ││   ││   ││   ││         ││   │
+└──────┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└─────────┘└───┘
+┌─────────┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌──────┐┌───┐┌───┐
+│         ││   ││   ││   ││   ││   ││   ││   ││   ││   ││   ││      ││   ││   │
+└─────────┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└──────┘└───┘└───┘
+┌────┐┌────┐┌────┐┌──────────────────────────────┐┌────┐┌────┐  ┌───┐┌───┐┌───┐
+│    ││    ││    ││                              ││    ││    │  │   ││   ││   │
+└────┘└────┘└────┘└──────────────────────────────┘└────┘└────┘  └───┘└───┘└───┘
+*/
+    /*  Row:    0        1        2        3        4        5        6        7        8        9        10       11       12       13       14     */
+    [_FN3]  = LAYOUT(
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+                _______, _______, _______,                   _______,                            _______, _______, _______, _______, _______
+            ),
```

Comment:
```suggestion
            )
```

Reply:
Can I keep this? (and in other keymaps). Because I want to put like 2 or 3 layers of Fn

Reply:
Nope, keymaps need to be minimal.